x-pack/plugins/secops/public/containers/hosts/index.tsx

```diff
@@ -103,7 +103,7 @@ const HostsComponentQuery = pure<HostsProps>(
 );
 
 const mapStateToProps = (state: State) => {
-  const limit = defaultTo(2, hostsSelector(state));
+  const limit = hostsLimitSelector(state);
```

Comment:
Optional: I think you can do a simple const mapStateToProps = (state: State) => hostsLimitSelector(state) at this point since it returns the same named property/key